platform-controller/src/test/java/org/hobbit/controller/docker/ContainerAccessByHostnameTest.java

```diff
@@ -20,6 +20,7 @@
 import static org.junit.Assert.assertNotNull;
 
 import com.spotify.docker.client.messages.ContainerConfig;
+import org.hobbit.controller.containers.docker.ContainerManagerImpl;
```

Comment:
When the tested class moves into a different package, the test classes should move, too.